### PR TITLE
Update node version for lambdas

### PR DESCRIPTION
**Pull request by KaliedaRik**
## What does this change?
Node version 8.x is no longer supported by AWS lambda service. This PR updates the code that generates the lambda code to set the node version to 12.x

### CODE REVIEW

File: lambdas/build.sh [hunk 1]
```diff
@@ -1,11 +1,7 @@
 #!/bin/bash
 
-#Installing node 6.10
-NODE_VERSION="6.10"
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
-nvm install ${NODE_VERSION}
-nvm use ${NODE_VERSION}
```
Comment (KaliedaRik) [inline lambdas/build.sh:8]: Do we need to keep the `nvm install` and `nvm use` instructions? Just not specify the node version?

**Comment by KaliedaRik** [comment]
I have concerns about yarn lock and brewfile lock files deciding to update themselves - I may have regenerated them accidentally. Advice?

File: lambdas/build.sh (hunk 1 above)
Comment (tomrf1) [reply lambdas/build.sh:8]: yes it should still run those commands, and use the `.nvmrc` for the version

File: nginx/Brewfile.lock.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (tomrf1) [inline nginx/Brewfile.lock.json:1]: I don't think we want this?!

Comment (KaliedaRik) [reply nginx/Brewfile.lock.json:1]: Should I delete both lock files, or just the Brewfile.lock file?

Comment (tomrf1) [reply nginx/Brewfile.lock.json:1]: updating the yarn.lock isn't a bad thing, I think it's fairly safe to leave in

Comment (MaelGNM) [reply nginx/Brewfile.lock.json:1]: yarn.lock should be kept in
> All yarn.lock files should be checked into source control. This allows Yarn to install the same exact dependency tree across all machines, whether it be your coworker’s laptop or a CI server.
(from [yarn documentation](https://classic.yarnpkg.com/en/docs/yarn-lock/))

Don't know about the brew one though

File: lambdas/build.sh (hunk 1 above)
Comment (KaliedaRik) [reply lambdas/build.sh:8]: Done

File: nginx/Brewfile.lock.json (hunk 2 above)
Comment (KaliedaRik) [reply nginx/Brewfile.lock.json:1]: Brewfile lock deleted. Yarn lock kept

**Review by MaelGNM — APPROVED** [review]
LGTM!

**Review by tomrf1 — APPROVED** [review]
great!

**Pull request merged**